.github/workflows/e2e-test-playwright.yml

```
@@ -0,0 +1,51 @@
+                  # the old cache within 7 days 👌
+                  # Note: don't change the `actions/setup-node@v1` or any other dependency version by error
+                  path: ~/.npm
+                  key: ${{ runner.os }}-build-v1.3-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+                  restore-keys: |
+                      ${{ runner.os }}-build-v1.3-${{ env.cache-name }}-
+                      ${{ runner.os }}-build-v1.3-
+                      ${{ runner.os }}-
+              env:
+                  cache-name: cache-node-modules
+            - name: Use Node.js 16.x
+              uses: actions/setup-node@v1
```

actionlint (1.7.7)

the runner of "actions/setup-node@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)